cpp/core/memory/ColumnarBatch.cc

```diff
@@ -118,19 +118,23 @@ std::shared_ptr<ColumnarBatch> CompositeColumnarBatch::create(std::vector<std::s
     }
     numColumns += batch->numColumns();
   }
-  return std::shared_ptr<ColumnarBatch>(new CompositeColumnarBatch(numColumns, numRows, batches));
+  return std::shared_ptr<ColumnarBatch>(new CompositeColumnarBatch(numColumns, numRows, std::move(batches)));
```

Comment:
👍 

Any performance improvement observed from your side?

Reply:
It seems Composite batch is only used by global sort. I did not see an obvious perfermance change.